tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/PackageUpdate/ILanguagePackageUpdate.cs

```diff
@@ -0,0 +1,33 @@
+using Azure.Sdk.Tools.Cli.Models.Responses.Package;
+
+namespace Azure.Sdk.Tools.Cli.Services;
+
+/// <summary>
+/// Service interface for language-specific package operations including metadata updates, changelog content updates, and version updates.
+/// </summary>
+public interface ILanguagePackageUpdate
+{
+    /// <summary>
+    /// Updates the package metadata content for a specified package.
+    /// </summary>
+    /// <param name="packagePath">The absolute path to the package directory.</param>
+    /// <param name="ct">Cancellation token for the operation.</param>
+    /// <returns>A response indicating the result of the metadata update operation.</returns>
+    Task<PackageOperationResponse> UpdateMetadataAsync(string packagePath, CancellationToken ct);
+
+    /// <summary>
+    /// Updates the changelog content for a specified package.
+    /// </summary>
+    /// <param name="packagePath">The absolute path to the package directory.</param>
+    /// <param name="ct">Cancellation token for the operation.</param>
+    /// <returns>A response indicating the result of the changelog update operation.</returns>
+    Task<PackageOperationResponse> UpdateChangelogContentAsync(string packagePath, CancellationToken ct);
+
+    /// <summary>
+    /// Updates the version for a specified package.
+    /// </summary>
+    /// <param name="packagePath">The absolute path to the package directory.</param>
+    /// <param name="ct">Cancellation token for the operation.</param>
+    /// <returns>A response indicating the result of the version update operation.</returns>
+    Task<PackageOperationResponse> UpdateVersionAsync(string packagePath, CancellationToken ct);
```

Comment:
May I know is this PR completed? why the `UpdateVersionAsync` method not have the parameters defined in the word format design doc? `sdkVersion` and `sdkReleaseType` is missing.
<img width="768" height="995" alt="image" src="https://github.com/user-attachments/assets/de9a60e6-e697-4c12-9c53-b7348bdd1bad" />

Reply:
The other two tools will be in separate PRs.